coordinator/src/main/scala/filodb.coordinator/queryengine2/MultiPodPlanner.scala

```diff
@@ -0,0 +1,17 @@
+package filodb.coordinator.queryengine2
+
+import filodb.query.{LogicalPlan, QueryContext}
+import filodb.query.exec.ExecPlan
+
+/**
+  * MultiPodPlanner is responsible for breaking down queries into separate subqueries,
```

Comment:
Perhaps remove this for now and move the comments to where you intend to use this.